mitodl/micromasters
# CMS: Remove external program url

#### What are the relevant tickets?
Fix #3728

#### What's this PR do?
Removes the external_program_url field for ProgramPage in cms.

#### How should this be manually tested?
Login to cms and make sure you don't see the field when editing a ProgramPage and make sure everything works.
